### ISSUE TITLE
hacs is completely uninstalled after uninstall an integration

### ISSUE THREAD

**Issue by engpedroamorim** [hacs/integration#1470]
## Installation details

Integration version: | 1.4.1
Frontend version: | 202018081656
Repositories: | 607
Installed repositories: | 9

## Describe the issue

if i uninstall accuweather instead of just removing this specific repository it removes the all Hacs instalation

### Steps to reproduce
hacs > integrations > accuweather > options: uninstall

**Comment by ludeeus** [comment]
Should possible if you are really running 1.4.1.
Attach logs to prove it.

**Comment by ludeeus** [comment]
I have no accuweather integration, but I installed and removed 3 others, what you are describing did not happen.

**Comment by engpedroamorim** [comment]
2020-08-18 17:37:25 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant.
2020-08-18 17:37:31 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for rpi_power which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant.
2020-08-18 17:37:33 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for authenticated which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant.
2020-08-18 17:37:48 ERROR (MainThread) [custom_components.hacs.factory] bieniu/ha-accuweather - Repository is archived.
2020-08-18 17:38:16 ERROR (MainThread) [custom_components.hacs.api.repository] Could not uninstall

**Comment by engpedroamorim** [comment]
Hacs shows a pending update for AccuWeather

Installed version: b89c55d
Update pending
Available version: 829ff80

Remember that you need to restart Home Assistant before changes to integrations (custom_components) are applied.

**Comment by ludeeus** [comment]
enable debug logs for HACS as instructed in the issue template.

**Comment by johntdyer** [comment]
Pretty sure this happened to me as well, havent been able to reproduce yet but my custom_compoonents folder disappears out of the blue when I uninstalled test/test

**Comment by ludeeus** [comment]
You don't have test/test @johntdyer upgrade to 1.4.1

**Comment by engpedroamorim** [comment]
cant upload the log because answer cant be longer then 65536 characters

**Comment by ludeeus** [comment]
That's fine, I got your mail.
And nothing there indicates that it removed anything.
It tried, but failed.
Is HACS gone now?

**Comment by markusmazurczak** [comment]
I have had the same issue. It occured after I did the following:

I updated HACS (not able to say from which to which version anymore) but I did not rebootet. After that update I updated HA to 0.114.0. After that reboot HACS was gone.

@johntdyer Gave me the hint to look for that `custom_components` folder which was deleted. I recreated that blank folder (`config/custom_components/hacs`) downloaded and copied the files from the zip file from here `https://github.com/hacs/integration/releases/tag/1.4.1`.

After the next reboot HACS appeared again and my everything else worked like charm.
